templates/common/openstack/units/afterburn-hostname.service.yaml

```diff
@@ -3,6 +3,9 @@ enabled: true
 contents: |
   [Unit]
   Description=Afterburn Hostname
```

Comment:
I think we'll also want to block services relying on Networking being up, same as [what GCP did](https://github.com/openshift/machine-config-operator/blob/0e66c5d6124440d298c6014a5372fe3f0f136c8e/templates/common/gcp/units/gcp-hostname.service.yaml#L8-L9).

Reply:
It's already done: https://github.com/openshift/machine-config-operator/blob/master/templates/common/openstack/units/afterburn-hostname.service.yaml#L13